packages/gatsby-cli/package.json

```diff
@@ -55,6 +54,9 @@
     "ink": "^2.0.5",
     "ink-spinner": "^3.0.1"
   },
+  "peerDependencies": {
```

Comment:
Let's remove it entirely? `gatsby-cli` doesn't actually need `react`

Reply:
I see `react` imports in this directory: https://github.com/gatsbyjs/gatsby/tree/82848c9614933f8dcb97822e4723121fe340365e/packages/gatsby-cli/src/reporter/reporters/ink

That's why I didn't remove it entirely.

Reply:
Oh never mind I missed that 🤦‍♂ Great catch!

Reply:
Moving it to peerDependencies will be problematic then because when installing `gatsby-cli` globally, peer deps won't be installed and this will break, right?

Reply:
:sweat: Yeah, you are right.

I don't see any workaround currently. Should I close this PR?

Reply:
Yeah, let's close for now